src/main/java/frc/robot/subsystems/intake/Command/Feed.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.subsystems.intake.Command;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.intake.Intake;
+
+public class Feed extends CommandBase {
+    private Intake intake = Intake.getINSTANCE();
+    private double power;
+
+    public Feed(Intake intake, double power) {
+        addRequirements(intake);
+        this.power = power;
+    }
+
+    /**
+     * open the retractor for cubes
+     */
+    @Override
+    public void initialize() {
+        intake.openRetractor();
+    }
+
+    /**
+     * start the motor of the Intake
+     */
+    @Override
+    public void execute() {
+        intake.setPower(power);
+    }
+
```

Comment:
Where is the `isFinished()` method?

Reply:
Went to get some milk

Reply:
JK

Reply:
we will combine the beamBreaker with the intake command in commndgroup

Reply:
A sensor is passive, you can't put it in a command because it ca'nt do anything.